frontend/src/app/shared/components/checkbox/checkbox.component.sass

```diff
@@ -14,6 +14,9 @@ input[type="checkbox"]
   position: relative
   margin-right: 5px
 
+input[type="checkbox"]:hover
```

Comment:
No need to add "hover', it will always be pointer

Reply:
done